README.md

```diff
@@ -52,6 +52,17 @@ Usage and commands available:
 ./ccd
 ```
 
+### Setting up environment variables
+Environment variables for CCD-Data-Store-API and CCD-Definition-Store-API can be done by executing the following script.
```

Comment:
Please remove dashes in natural language uses.
CCD-Data-Store-API and CCD-Definition-Store-API --> CCD Data Store API and CCD Definition Store API

Reply:
Removed the dashes.